src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
+  max-width: 1024px;
```

Comment:
I would recommend to add media queries to handle different screen width and modify max-width property.

Example: 

@media only screen and (min-width: 992px) {
     max-width: 1000px;
}

@media only screen and (min-width: 1200px) {
   max-width: 1400px;
}

Note: you also can create mixins to reuse the same media queries